core/digital-twin-lib/src/DAppTwin.ts

```diff
@@ -92,25 +95,19 @@ class DAppTwin extends DigitalTwin {
    * Reset containers object and reload all container definitions.
    */
   private async ensureContainers(): Promise<void> {
-    this.containers = { };
-    this.containerContracts = { };
-
     // transform twinEntries to containers object, so all
     const twinEntries = await this.getEntries();
-    this.containerKeys = Object.keys(twinEntries);
-    await Promise.all(this.containerKeys.map(async (key: string) => {
-      this.containers[key] = new DAppContainer(
-        this.vue,
-        this.runtime,
-        await twinEntries[key].value.getContractAddress(),
-      );
-
-      // load description, contractAddress, dispatcherStates
-      await this.containers[key].initialize();
-
-      // make contracts also directly accessible
-      this.containerContracts[this.containers[key].contractAddress] = this.containers[key];
+    await Promise.all(Object.keys(twinEntries).map(async (key: string) => {
+      const address = await twinEntries[key].value.getContractAddress();
+
+      // only load container, if it wasn't initialized before (e.g. when it was initially loaded)
```

Comment:
What is meant by "initialized" here? According to the `setupContainer` method it's possible that this.containerContracts[address] is set but, `container.initialize` is not called.

Reply:
Its a misleading comment. Is connected to the `loadBaseInfo` function and if no `DAppContainer` instance was added to the twin.